### PR TITLE
DBSCAN notebook Twitter exmaple

### PR DESCRIPTION
**Pull request by Er1cCheng**

### CODE REVIEW

**Comment by Er1cCheng** [comment]
build

File: notebooks/dbscan.ipynb [hunk 1]
```diff
@@ -481,6 +481,93 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import h5py\n",
+    "\n",
+    "# Consider using multiple GPUs given the large datasize\n",
+    "with h5py.File('twitterSmall.h5.h5', 'r') as f: \n",
```
Comment (eordentlich) [inline notebooks/dbscan.ipynb:528]: Can the file be converted to parquet format and copied to /dbfs ?

File: notebooks/dbscan.ipynb [hunk 2]
```diff
@@ -481,6 +481,93 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Full dataset\n",
+    "# !curl --output twitter.h5.h5 https://b2share.eudat.eu/api/files/189c8eaf-d596-462b-8a07-93b5922c4a9f/twitter.h5.h5\n",
+    "\n",
+    "# Partial small dataset\n",
+    "!curl --output twitterSmall.h5.h5 https://b2share.eudat.eu/api/files/189c8eaf-d596-462b-8a07-93b5922c4a9f/twitterSmall.h5.h5"
```
Comment (lijinf2) [inline notebooks/dbscan.ipynb:509]: Do we always get the same dataset? May need to keep a copy of it locally or on the cloud storage (e.g. the /dbfs mentioned).

**Review by lijinf2 — COMMENTED** [review]
Overall looks good!

**Comment by Er1cCheng** [comment]
build

**Review by eordentlich — APPROVED** [review]
👍

**Pull request merged**